src/Diagnostics/atmos_default.jl

```diff
@@ -3,16 +3,70 @@ using ..Atmos: thermo_state, turbulence_tensors
 using ..Mesh.Topologies
 using ..Mesh.Grids
 using ..MoistThermodynamics
-using CLIMAParameters.Atmos.SubgridScale: inv_Pr_turb
 using LinearAlgebra
 
-Base.@kwdef mutable struct AtmosCollectedDiagnostics
+Base.@kwdef mutable struct AtmosDefaultCollectedDiagnostics
     zvals::Union{Nothing, Array} = nothing
     repdvsr::Union{Nothing, Array} = nothing
 end
-const CollectedDiagnostics = AtmosCollectedDiagnostics()
+const AtmosDefaultCollected = AtmosDefaultCollectedDiagnostics()
 
-include("diagnostic_vars.jl")
+function vars_diagnostic(FT)
+    @vars begin
+        # state and functions of state
+        rho::FT                 # ρ
+        u::FT
+        v::FT
+        w::FT
+        qt::FT                  # q_tot
+        et::FT                  # e_tot
+        ql::FT                  # q_liq
+        thd::FT                 # θ
+        thl::FT                 # θ_liq
+        thv::FT                 # θ_v
+        ei::FT                  # e_int
+        hm::FT
+        ht::FT
+        qt_sgs::FT
+        ht_sgs::FT
+
+        # vertical fluxes and covariances
+        cov_w_rho::FT           # w′ρ′
+        cov_w_u::FT             # w′u′
+        cov_w_v::FT             # w′v′
+        cov_w_qt::FT            # w'q_tot'
+        w_qt::FT                # w q_tot
+        cov_w_ql::FT            # w′q_liq′
+        cov_w_qv::FT            # w′q_vap′
+        cov_w_thd::FT           # w′θ′
+        cov_w_thl::FT           # w′θ_liq′
+        cov_w_thv::FT           # w′θ_v′
+        cov_w_ei::FT            # w′e_int′
+        cov_qt_thl::FT          # q_tot′θ_liq′
+        cov_qt_ei::FT           # q_tot′e_int′
+
+        # variances
+        var_u::FT               # u′u′
+        var_v::FT               # v′v′
+        var_w::FT               # w′w′
+        var_qt::FT              # q_tot′q_tot′
+        var_ei::FT              # e_int′e_int′
+        var_thl::FT             # thl′thl′
+
+        # skewness
+        w3::FT                  # w′w′w′
+
+        # turbulent kinetic energy
+        tke::FT
```

Comment:
As it is now there is a potential bug in the calculation of tke. 
Since tke can be obtained from other quantities that are written to file, should be just omit it from output?